### PR TITLE
media : fix wrong data type for datasource

### PR DESCRIPTION
**Pull request by jsdosa**
fix wrong return type of datasource

Signed-off-by: jaesick.shin <jaesick.shin@samsung.com>

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [f6c1686324e98a9237225d7dd2799db1b2f7242c] - Code Rule Check Result:
framework/include/media/DataSource.h:66: ERROR: [SPC_M_OPR] need consistent spacing around '&' (ctx:VxW)
framework/include/media/DataSource.h:66: ERROR: [SPC_M_OPR] spaces required around that '=' (ctx:VxV)

**Comment by juitem** [comment]
I'd rather suggest use unsigned char for 'channel'

File: framework/include/media/DataSource.h [hunk 1]
```diff
@@ -91,14 +91,14 @@ class DataSource
 	 * @return The channel count of the stream data.
 	 * @since TizenRT v2.0 PRE
 	 */
-	int getChannels();
+	unsigned short getChannels();
 	/**
```
Comment (juitem) [inline framework/include/media/DataSource.h]: 16bit also looks too much for channels,
Would you please consider to use unsigned char?

Comment (jsdosa) [reply framework/include/media/DataSource.h]: yes, you right.

Comment (jsdosa) [reply framework/include/media/DataSource.h]: @juitem  fixed it.

**Comment by seinfra** [comment]
Target : [961b87c572fa77237af745d48b11779769e3a1ca] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [961b87c572fa77237af745d48b11779769e3a1ca] - Code Rule Check Result:
framework/include/media/DataSource.h:66: ERROR: [SPC_M_OPR] need consistent spacing around '&' (ctx:VxW)
framework/include/media/DataSource.h:66: ERROR: [SPC_M_OPR] spaces required around that '=' (ctx:VxV)

**Pull request merged**